boa/contracts/vyper/vyper_contract.py

```diff
@@ -126,6 +126,26 @@ def at(self, address: Any) -> "VyperContract":
 
         return ret
 
+    def verify(self, address: Address, explorer, license=None) -> None:
+        """
+        Verifies the Vyper contract on a block explorer.
+        :param address: The address of the contract.
+        :param explorer: The block explorer to use for verification.
+        :param license: The license to use for the contract. Defaults to "none".
+        """
+        if self.filename == "<unknown>":
+            raise ValueError(
+                "Cannot verify a contract without a filename. "
+                "Please use `boa.load` and save the contract to a file, for now."
+            )
+        explorer.verify(
+            address=address,
+            standard_json=build_solc_json(self.compiler_data),
```

Comment:
ah i see, being a method here kind of makes sense bc the contract also encapsulates compiler data